.github/workflows/CICD.yml

```diff
@@ -1,18 +1,61 @@
 name: CICD
-
-on: [push, pull_request]
-
-env:
-  CARGO_TERM_COLOR: always
-
+on: [pull_request, push]
 jobs:
-  build:
-
+  test:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        toolchain: [stable, nightly]
+    continue-on-error: ${{ matrix.os == 'windows-latest' && matrix.toolchain == 'nightly' }}
```

Comment:
Is it possible to always continue on errors?

Reply:
Sure no problem. 

According to the docs, that would mean a PR is valid even in case of an error, though. So, if there is a real error in one of the other test cases, it would be considered okay for merge.

If that side effect is okay with you, I'll mark all tests to continue on error :+1: